### PR TITLE
Fix containerd logo link.

### PR DESCRIPTION
**Pull request by Random-Liu**
The image was moved by https://github.com/containerd/containerd/pull/2342.

Use a link to a specific tag.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

**Comment by Random-Liu** [comment]
We should cherrypick this to `release/1.0` to fix the image link there as well.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # cri
 <p align="center">
 <img src="https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png" width="50" height="50">
-<img src="https://github.com/containerd/containerd/blob/master/docs/images/containerd-dark.png" width="200" >
+<img src="https://github.com/containerd/containerd/blob/v1.1.0/docs/images/containerd-dark.png" width="200" >
```
Comment (mikebrow) [inline README.md]: it moved to the docs/static/img directory... we should use  "https://github.com/containerd/containerd/blob/master/docs/static/img/containerd-dark.png"

Comment (Random-Liu) [reply README.md]: If we use master, we may still have the problem in the future.
Let's use a tagged release, so it won't change.

**Comment by Random-Liu** [comment]
Close in favor of https://github.com/containerd/cri/pull/775.

**Pull request closed without merging**